CRM/Activity/BAO/Activity.php

```diff
@@ -1685,7 +1685,7 @@ public static function getContactActivity($contactId) {
    * @return bool|NULL
    */
   public static function addActivity(
-    &$activity,
+    $activity,
```

Comment:
this isn't changed + it doesn't make sense with objects to pass by ref anyway - good change